### PR TITLE
mohammad/metatrader_pages

### PR DESCRIPTION
**Pull request by binary-hamedanchi**

### CODE REVIEW

File: src/templates/static/platforms.html.tt [hunk 1]
```diff
@@ -53,138 +53,138 @@
...
-                        <div class="gr-row gr-padding-30 gr-parent">
-                            <div class="gr-6 gr-12-m">
-                                <img class="responsive" src="[% request.url_for('images/pages/platforms/mt5/interface.svg') %]" alt="" />
+                        <div class="gr-padding-10 fill-bg-color">
+                            <div class="container gr-padding-30">
+                                <h2 class="center-text">[% l("Enhanced Trading with MetaTrader 5") %]</h2>
+                                <div class="gr-row center-text enhanced">
+                                    [% INCLUDE enhanced_items image='trade'      text=l("Trade Forex and CFD asset classes with an award winning broker") %]
+                                    [% INCLUDE enhanced_items image='leverage'   text=l("Trade the Forex markets with leverage up to 100:1, and Volatility Indices with leverage up to 500:1") %]
+                                    [% INCLUDE enhanced_items image='demo'       text=l("Open a free Demo account. Upgrade to a Real account by depositing funds thru your Binary.com account cashier") %]
```
Comment (kellybinary) [inline src/templates/static/platforms.html.tt]: minor typo here: `thru` should be through :-)

File: src/templates/static/platforms.html.tt [hunk 2]
```diff
@@ -53,138 +53,138 @@
...
+                        </div>
+                        <div class="gr-row gr-padding-30">
+                            [% INCLUDE platform image_class='gr-8 gr-5-p gr-7-m' image='webtrader' header=l("Binary Webtrader") description=l("Advanced binary options trading interface")
+                                text=l("Monitor the movements of your favourite assets and markets at the same time.")
+                                url='https://webtrader.binary.com' target='_blank' button_text=l("Try Webtrader") %]
+                            <div class="gr-2"></div>
+                            [% INCLUDE platform image_class='gr-12 gr-7-p gr-10-m' image='nextgen' header=l("Binary Next-Gen") description=l("Advanced trading app for web and mobile")
+                                text=l("Explore advanced trading features with the Next-Gen app for web and mobile.")
+                                url='https://app.binary.com' target='_blank' button_text=l("Try Next-Gen")
+                                app_store_url='https://itunes.apple.com/app/apple-store/id1134884301?pt=117911266&ct=platforms_page&mt=8'
```
Comment (kellybinary) [inline src/templates/static/platforms.html.tt]: should we remove the link to next-gen apple store? if i remember correctly, the next-gen app was removed from the apple store

File: src/templates/static/platforms.html.tt [hunk 3]
```diff
@@ -53,138 +53,138 @@
...
+                                                <li>[% l("Safari") %]</li>
+                                                <li>[% l("Firefox") %]</li>
+                                                <li>[% l("Edge") %]</li>
+                                                <li>[% l("Opera") %]</li>
+                                            </ul>
+                                        </div>
                                     </div>
+                                    <p class="center-text">
+                                        <a class="button button-secondary full-width" href="[% request.url_for('metatrader/download') %]">
+                                            <span>[% l("Lear more") %]</span>
```
Comment (kellybinary) [inline src/templates/static/platforms.html.tt]: minor typo here too: `Learn` :-)

File: src/templates/static/platforms.html.tt (hunk 2 above)
Comment (ashkanx) [reply src/templates/static/platforms.html.tt]: yes. please remove iOS links to next-gen and tick trade

File: src/templates/static/platforms.html.tt (hunk 1 above)
Comment (binary-hamedanchi) [reply src/templates/static/platforms.html.tt]: Fixed, thanks.

**Pull request merged**